src/components/toast.js

```diff
@@ -10,13 +10,17 @@ import { concatTargetColor } from '../utils/concat-color'
 const Toast = forwardRef(({
   classes,
   variant,
+  display,
+  setDisplay,
```

Comment:
would suggest not to include `setDisplay` as a prop, users can handle the `display -> true/false` externally, and via `onClose`